tests/UI/campaigns/functional/API/01_clientCredentialGrantFlow/02_externalAuthServer/01_authorizationEndpoint.ts

```diff
@@ -0,0 +1,134 @@
+import api from '@utils/api';
+import keycloakHelper from '@utils/keycloakHelper';
+import testContext from '@utils/testContext';
+
+import {expect} from 'chai';
+import {APIRequestContext, request} from 'playwright';
+import {installKeycloak, createKeycloakClient, uninstallKeycloak} from '@commonTests/API/keycloak';
```

Comment:
=> 'installKeycloak' is defined but never used  
Please fix the lint errors

Reply:
Fixed